### ISSUE TITLE
Questions about `Changed`

### ISSUE THREAD

**Issue by lumynou5** [olivierlacan/keep-a-changelog#636]
In my opinion, these categories have two kinds: note and warning. (The words are picked from compiler message. I just don't know better words.)

For example, `Added` is a "note". User doesn't need to know what's added to upgrade safely. `Fixed` is also a "note". Although a fix might change behavior, the behavior is incorrect, unexpected, and/or different from the documentation.

In the other hand, `Deprecated` and `Removed` are "warnings". They indicate to user preparation is needed.

However, `Changed` seems very wide, including various big and small changes. Changes to functionality, behavior, and interface are "warnings" without a doubt, but there are also other kinds of changes in this category. A common one is optimization, which doesn't added new thing, fix a bug, or change any behavior but improve the performance. Optimization seems not changes any public interface but it does matter and users care, unlike really "invisible" ones such as changes to unit test and CI.

It seems inconvenient to find a change fast because `Changed` has a such vary importance, and makes bumping semantic version numbers harder—other categories are obvious; `Fixed` increases patch version, `Added` increases minor version, and `Removed` increases major version, etc.

Do I understand `Changed` wrong? Where should I put optimization?

**Comment by william-andersson** [comment]
I agree. Maybe Changes could be split up into Changed and Updated? I regularly make changes to the code that does not change existing functionality... Maybe there could be a type 'note' or 'cleanup' section too?

**Comment by lumynou5** [comment]
@william-andersson 

That could be a solution, and looks like:
- `Changed`
  "Big changes", breaking changes fall into this category.
- `Updated`
  This category includes smaller changes that are safe to upgrade, such as a better performance and a slight change to application UI.

The only problem is its name. `Updated` is as abstract and obscure as `Changed`, people might mix up.

**Comment by william-andersson** [comment]
You are absolutely right. Maybe `Refactor` would be a better word for it?

**Comment by DJCrashdummy** [comment]
i'm not sure if it should be split... and even less about the names. but i'm pretty sure either way, it will confuse some people. 😑

but interestingly - although a different topic - i similarly classified the sections as "warning" and "note" without mentioning it (but by ranking them): https://github.com/olivierlacan/keep-a-changelog/issues/458#issuecomment-2459808902
...and yes, `Changed` was/is a pretty hard one to crack. 😜

**Comment by swashdev** [comment]
I think splitting hairs in the terminology of `Changed`, `Updated`, `Refactored`, et cetera, is potentially confusing for end-users, but I do agree that there's an important distinction between _feature_ changes and _technical_ changes.  Most end-users only care about changes to functionality, but techies who depend on downstream support or who are contributing to an open-source project will want to know the technical details of changes happening under the hood.

For some projects in the past I've had separate sections for `Changes` (as in gameplay changes) and `Code Changes` for this reason.  To be more broadly applicable, you can use `Changes` to refer to feature changes or functionality changes that most end-users care about, and have a separate section for `Technical Changes` which reference changes to technical specifications and functionality which aren't applicable to non-technically-inclined users.

You could have these be sub-sections within the `Changed` section, but to avoid confusing non-technical end-users it may be best to have this in a separate section, in which case `Refactored` might be a more applicable title, since tech-savvy users will be more likely to know what that means, but since I have limited experience in such matters I'll leave that there.

**Comment by lumynou5** [comment]
> For some projects in the past I've had separate sections for Changes (as in gameplay changes) and Code Changes for this reason. To be more broadly applicable, you can use Changes to refer to feature changes or functionality changes that most end-users care about, and have a separate section for Technical Changes which reference changes to technical specifications and functionality which aren't applicable to non-technically-inclined users.

That's a good ideal (though not suitable for every kind of projects). So for example, in a game we can have `Changes` for the gameplay and `Technical Changes` for mod interface changes, etc. while the former is not considered breaking changes and the latter is.
